pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -103,9 +103,8 @@ def log_metrics(self, metrics: Dict[str, _METRIC], step: Optional[int] = None) -
             step = self.trainer.global_step
 
         # log actual metrics
-        if self.trainer.is_global_zero:
-            self.trainer.logger.agg_and_log_metrics(scalar_metrics, step=step)
-            self.trainer.logger.save()
+        self.trainer.logger.agg_and_log_metrics(scalar_metrics, step=step)
```

Comment:
Mind sharing the reasons why we would want all processes to log ?

Reply:
@tchaton I updated the PR description. The motivation here is to enable implementations of loggers that actually can lon every rank. The existing loggers we have internally already do have checks to log only on rank 0.